fix link to Example.ipynb (#123)
src-d/spark-api: docs/usage/commands.md

If you want to completely uninstall `sourced` you must also delete the `~/.sourced` directory.

### sourced logs

Show logs from components.

> If it's passed `--follow`, the logs are shown as they are logged till you exit with `Ctrl+C`.

Reviewer: ```suggestion
If `--follow` is used the logs are shown as they are logged until you exit with `Ctrl+C`.
```

> It can be passed only some component names, to see only its logs.

Reviewer: ```suggestion
You can optionally pass component names to see only their logs.
```
